Lib/test/test_types.py

```diff
@@ -772,6 +775,31 @@ def test_union_parameter_chaining(self):
         self.assertEqual((list[T] | list[S])[int, T], list[int] | list[T])
         self.assertEqual((list[T] | list[S])[int, int], list[int])
 
+    def test_union_pickle(self):
+        alias = list[T] | int
+        s = pickle.dumps(alias)
+        loaded = pickle.loads(s)
+        self.assertEqual(alias.__args__, loaded.__args__)
+        self.assertEqual(alias.__parameters__, loaded.__parameters__)
```

Comment:
Those are good checks 👍🏻
I'd also put a plain `self.assertEqual(alias, loaded)`. At the moment it's redundant with comparing `__args__` but this might not be true later, so this will keep future-proof.

Reply:
Thanks, I added `self.assertEqual(alias, loaded)` line to test.